### PR TITLE
* add support for customizing icons

### PR DESCRIPTION
**Pull request by Olli**

### CODE REVIEW

File: src/baguetteBox.js [hunk 1]
```diff
@@ -48,7 +48,10 @@
             afterShow: null,
             afterHide: null,
             onChange: null,
-            overlayBackgroundColor: 'rgba(0,0,0,.8)'
+            overlayBackgroundColor: 'rgba(0,0,0,.8)',
+            closeX: closeX,
+            leftArrow: leftArrow,
+            rightArrow: rightArrow,
```
Comment (feimosi) [inline src/baguetteBox.js]: Please add `Icon` suffix to these options:

```suggestion
            rightArrowIcon: rightArrow
```

File: src/baguetteBox.js [hunk 2]
```diff
@@ -348,8 +351,7 @@
             return;
         }
         currentGallery = gallery;
-        // Update gallery specific options
-        setOptions(userOptions);
```
Comment (feimosi) [inline src/baguetteBox.js]: Why `setOptions` is being removed?

**Comment by feimosi** [comment]
@Olli thanks for your contribution! Please fix the linter errors:

https://travis-ci.org/feimosi/baguetteBox.js/jobs/445689660

File: src/baguetteBox.js (hunk 2 above)
Comment (Olli) [reply src/baguetteBox.js]: Don't know anymore. Sry

**Comment by feimosi** [comment]
For anyone subscribed to this issue, this functionality has just been released based on #245

**Pull request closed without merging**